Lombiq.ChartJs/Models/DataLabelAlignmentConfiguration.cs

```diff
@@ -54,7 +55,7 @@ public class FontStyle
         [JsonIgnore]
         public bool IsBold { get; set; }
 
-        [JsonProperty("weight")]
+        [JsonPropertyName("weight")]
         public string Weight
         {
             get => IsBold ? "bold" : "normal";
```

Comment:
I know, this is not done by you, but `"bold"` is a good candidate to be a `const` string. Also `"center"`, `"start"`, and `"end"`.

Reply:
I'd avoid unrelated refactoring in this issue, since it won't be merged into `dev` until OC 2.0 is out.